ddtrace/propagation/_database_monitoring.py

```diff
@@ -114,6 +117,10 @@ def _get_dbm_comment(self, db_span):
         if peer_hostname:
             dbm_tags[DBM_PEER_HOSTNAME_KEY] = peer_hostname
 
+        peer_service = db_span.get_tag(self.peer_service_tag)
+        if peer_service:
```

Comment:
Do we have defined behavior for `self.peer_service_tag = ""` ???

Reply:
its the `peer.service` tag